runtime/interpreter/value.go

```diff
@@ -2371,6 +2383,18 @@ func (v *ArrayValue) GetMember(interpreter *Interpreter, locationRange LocationR
 			},
 		)
 
+	case "reverse":
```

Comment:
Define a `const ArrayTypeReverseFunctionName = "reverse"` in `sema`, and then use it there and here

Reply:
Done.